<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HARDIFiberTrackingForm</class>
 <widget class="QWidget" name="HARDIFiberTrackingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>881</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="DataSelectionHLayout">
     <item>
      <layout class="QVBoxLayout" name="DataSelectionLabelVLayout">
       <item>
        <widget class="QLabel" name="DTIDataLabel">
         <property name="text">
          <string>HARDI Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="AIDataLabel">
         <property name="text">
          <string>Scalar Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="DataSelectionComboVLayout">
       <item>
        <widget class="QComboBox" name="HARDIDataCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="AIDataCombo"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Seed Points</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="seedList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::SinglePass</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="trackingMethodHLayout">
     <item>
      <widget class="QLabel" name="trackingMethodLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Method   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="trackingMethodCombo">
       <item>
        <property name="text">
         <string>Deterministic</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolBox" name="fiberTrackingToolbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>6</number>
     </property>
     <widget class="QWidget" name="trackingParametersPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>272</width>
        <height>456</height>
       </rect>
      </property>
      <attribute name="label">
       <string notr="true">Tracking Parameters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="parametersMinLengthLabel">
         <property name="text">
          <string>Minimum Fiber Length (mm) </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="parametersMinLengthHLayout">
         <item>
          <widget class="QSpinBox" name="parametersMinLengthSpinner">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="parametersMinLengthSlider">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="parametersMaxLengthLabel">
         <property name="text">
          <string>Maximum Fiber Length (mm)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="parametersMaxLengthHLayout">
         <item>
          <widget class="QSpinBox" name="parametersMaxLengthSpinner">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="parametersMaxLengthSlider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="parametersMinAILabel">
           <property name="text">
            <string>Min. Scalar Threshold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="parametersMinAISpin">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.150000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="parametersMaxAILabel">
           <property name="text">
            <string>Max. Scalar Threshold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="parametersMaxAISpin">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="parametersAngleLabel">
         <property name="text">
          <string>Maximum Fiber Angle (Degrees / Step)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="parametersAngleHLayout">
         <item>
          <widget class="QSpinBox" name="parametersAngleSpinner">
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="value">
            <number>45</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="parametersAngleSlider">
           <property name="maximum">
            <number>90</number>
           </property>
           <property name="value">
            <number>45</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="parametersStepSizeLabel">
           <property name="text">
            <string>Step Length (Voxels)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="parametersStepSizeSpinner">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.050000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="iterationlabel">
           <property name="text">
            <string>Iterations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="iterationSpinner">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="tresholdLabel">
           <property name="text">
            <string>ODF threshold</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="tresholdSpinner">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.500000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="tesselationLabel">
           <property name="text">
            <string>Tessellation order</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tesselationSpinner">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>8</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QCheckBox" name="useMaxFileCheck">
           <property name="text">
            <string>Use File for Maxima</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="cleanBox">
         <property name="text">
          <string>Clean ODF maxima</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dummyPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>98</width>
        <height>28</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Dummy Page</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="updateButton">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="overwriteCheck">
     <property name="text">
      <string>Overwrite existing fibers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>parametersAngleSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>parametersAngleSpinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>709</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>708</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parametersAngleSpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>parametersAngleSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>708</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>709</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parametersMaxLengthSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>parametersMaxLengthSpinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parametersMaxLengthSpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>parametersMaxLengthSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>589</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parametersMinLengthSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>parametersMinLengthSpinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>528</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>parametersMinLengthSpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>parametersMinLengthSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
